### PR TITLE
Update history.txt

### PR DESCRIPTION
**Pull request by nino2armando**

### CODE REVIEW

File: history.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 some new stuff
-
+sdfasdfasdf
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (nino2armando) [inline history.txt:5]: @nino-k this is testing for notifications

Comment (nino2armando) [inline history.txt:5]: @nino-k testing the polling

**Comment by Nino-K** [comment]
@nino-k another mention

**Comment by nino2armando** [comment]
@Nino-K test comment

**Comment by nino2armando** [comment]
@Nino-K testing

**Comment by nino2armando** [comment]
@Nino-K some new comments

**Comment by nino2armando** [comment]
@Nino-K hello world!

**Comment by nino2armando** [comment]
@Nino-K testing

**Comment by nino2armando** [comment]
@Nino-K polling test

**Comment by nino2armando** [comment]
@Nino-K test

**Comment by nino2armando** [comment]
@Nino-K called from intergration test, to test the interval

**Comment by nino2armando** [comment]
@Nino-K  called from intergration test, to test the interval

**Comment by nino2armando** [comment]
@Nino-K called from intergration test, to test the interval

**Comment by nino2armando** [comment]
@Nino-K testing

**Comment by nino2armando** [comment]
@Nino-K testing

**Comment by nino2armando** [comment]
@Nino-K testing

**Comment by nino2armando** [comment]
@Nino-K Testing

**Comment by nino2armando** [comment]
@Nino-K Testing

**Pull request closed without merging**